packages/features/ee/workflows/components/WorkflowDetailsPage.tsx

```diff
@@ -47,7 +48,13 @@ export default function WorkflowDetailsPage(props: Props) {
     [data]
   );
 
-  const addAction = (action: WorkflowActions, sendTo?: string, numberRequired?: boolean, sender?: string) => {
+  const addAction = (
```

Comment:
Consider turning this into a config object instead of a function with 5 fn params.